website/docs/MigrationV12.md

```diff
@@ -0,0 +1,60 @@
+---
+id: migration-v12
+title: Migration to 12.0
+---
+
+React Native Testing Library 12 introduces a handful of breaking changes compared to 11.x versions. We believe they were necessary to improve the experience using the library and help the users [fall into the pit of success](https://blog.codinghorror.com/falling-into-the-pit-of-success/) when writing meaningful tests. You will find migration instructions for each and every change described below.
+
+# Breaking changes
+
+## 1. `*ByRole` queries now return only accessibility elements
+`*ByRole` queries now return only accessibility elements, either explicitly marked with `accessible` prop or implicit ones where this status is derrived from component type itself (e.g `Text`, `TextInput`, `Switch`, but not `View`).
+
+You may need to adjust relevant components under test to make sure they pass `isAccessibilityElement` check.
+
+### Examples
+Let's assume we are using `getByRole("button")` query.
+
+Following elements will match:
+
+```ts
+// Explicit "accessible" prop for View
+<View accessible role="button" />
+
+// No need to "accessible" prop for Text, as it is implicitly accessible element.
+<Text role="button">Button</Text>
+```
+
+While following elements will not match:
+
+```ts
+// Missing "accessible" prop for View
+<View accessible={false} role="button" />
+
+// Explicit "accessible={false}" prop for View
+<View accessible={false} role="button" />
+
+// Explicit "accessible={false}" for Text,which is implicitly accessible element
+<Text accessible={false} role="button">Button</Text>
+```
+
+## 2. `*ByText`, `*ByDisplayValue`, `*ByPlaceholderText` queries now return host components
+`*ByText`, `*ByDisplayValue`, `*ByPlaceholderText` queries now return host components, as all other queries.
```

Comment:
I'd have linked the testing environment doc that explains what host components are